src/models/comments_model.ts

```diff
@@ -0,0 +1,28 @@
+import mongoose, { Document, Schema, ObjectId } from "mongoose";
+
+export interface ICommentDocument extends Document {
+  _id: ObjectId,
+  postID: ObjectId;
```

Comment:
From what I read, `ObjectId` is for mongoose's schema configuration only.
When you want to use it for typescript you should use `Types.ObjectId`.
So, the interface should use `Types.ObjectId` and the schema should use `Schema.Types.ObjectId`.
Change it and your import to: `import { Schema, Types, model } from "mongoose";`